flask_gcp_log_groups/background_thread.py

```diff
@@ -83,14 +83,14 @@ def _thread_main(self):
                     # all items we got back before quitting.
                 else:
                     if (item['message'] is  None):
-                      batch.log_text(None,  timestamp=item['timestamp'], labels=item['labels'], severity=item['severity'], trace=item['trace'], span_id=item['span_id'], http_request=item['http_request']) 
+                      batch.log_text(None,  timestamp=item['timestamp'], resource=item['resource'], labels=item['labels'], severity=item['severity'], trace=item['trace'], span_id=item['span_id'], http_request=item['http_request']) 
                     else:
                       try:
                           msg=ast.literal_eval(item['message'])
-                          batch.log_struct(msg,  timestamp=item['timestamp'], labels=item['labels'], severity=item['severity'], trace=item['trace'], span_id=item['span_id'], http_request=item['http_request'])
+                          batch.log_struct(msg,  timestamp=item['timestamp'], resource=item['resource'], labels=item['labels'], severity=item['severity'], trace=item['trace'], span_id=item['span_id'], http_request=item['http_request'])
                       except Exception as e:
                         #print("Error " + str(e))
-                        batch.log_text(item['message'],  timestamp=item['timestamp'], labels=item['labels'], severity=item['severity'], trace=item['trace'], span_id=item['span_id'], http_request=item['http_request']) 
+                        batch.log_text(item['message'],  timestamp=item['timestamp'], resource=['resource'], labels=item['labels'], severity=item['severity'], trace=item['trace'], span_id=item['span_id'], http_request=item['http_request']) 
```

Comment:
its 
```resource=item['resource']```

Reply:
Thanks for the PR.  

The ```resource``` is actually of

 - https://cloud.google.com/logging/docs/reference/v2/rest/v2/MonitoredResource
 - https://github.com/googleapis/google-cloud-python/blob/master/logging/google/cloud/logging/resource.py


so one possible usage would be on init to do something like

```python
from google.cloud.logging.resource import Resource
app = Flask(__name__)

_GLOBAL_RESOURCE = Resource(type='global', labels={})
my_resource = Resource("gce_instance", 
              labels={ "instanceId": "12345678901234",
                       "zone": "us-central1-a" 
                      }
            )

g = GCPHandler(app, parentLogName="request",
    childLogName="application",
    traceHeaderName='X-Cloud-Trace-Context',
    labels= {'foo': 'bar', 'baz': 'qux'},
    resource=my_resource)
g.setLevel(logging.INFO)
app.logger.addHandler(g)
```

or maybe just pass in a `dict` representing that

```python
g = GCPHandler(app, parentLogName="request",
    childLogName="application",
    traceHeaderName='X-Cloud-Trace-Context',
    labels= {'foo': 'bar', 'baz': 'qux'},
    resource= {
              "type": "gce_instance", 
              labels={ "instanceId": "12345678901234",
                            "zone": "us-central1-a" 
               }
    }
 )
g.setLevel(logging.INFO)

```
and then reconstructing the `Resource` object internally
---

I also realized its missing
 - https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#MonitoredResourceMetadata
---